docs/user-guide.md

```diff
@@ -416,14 +417,14 @@ triplets by using more complex motifs.
 
 <div data-lang="scala"  markdown="1">
 {% highlight scala %}
-import org.graphframes.examples
-val g: GraphFrame = examples.Graphs.friends  // get example graph
+import org.graphframes._
+val g = examples.Graphs.friends  // get example graph
 
 // Select subgraph based on edges "e" of type "follow"
 // pointing from a younger user "a" to an older user "b".
-val paths = g.find("(a)-[e]->(b)")
-  .filter("e.relationship = 'follow'")
-  .filter("a.age < b.age")
+val paths = g.find("(a)-[e]->(b)").
+  filter("e.relationship = 'follow'").
+  filter("a.age < b.age")
```

Comment:
Spark code generally have the `.` on the next line, for example
https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/ml/DecisionTreeExample.scala

Reply:
I see, yes, that could be helpful. is there another way for this though?
like
```
val paths = (g.find()
                     .filter())
```

or that is ugly?

Reply:
No, that's ok - I just pushed new commit, but used the `{}` to mark these as blocks of code...